src/tools/genpolicy/src/version.rs.in

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2020 Intel Corporation
```

Comment:
Might want to add a line for Microsoft if we modified this file from Intel: https://docs.opensource.microsoft.com/contributing/

Reply:
Good catch. TIL about `Portions Copyright`